src/sentry/models/deletedorganization.py

```diff
@@ -0,0 +1,17 @@
+from __future__ import absolute_import
+
+from django.db import models
+
+from sentry.db.models import sane_repr
+from sentry.models.deletedentry import DeletedEntry
+
+
+class DeletedOrganization(DeletedEntry):
+    name = models.CharField(max_length=64, null=True)
+    slug = models.SlugField(null=True)
```

Comment:
As a minor nit, there's no reason or value in actually using `SlugField` in any of these. SlugField simply provides an extra layer of validation with forms and has other properties. But since we're treating this as sort of a log, we can safely assume the data coming from the Organization or whatever table is already correct, and havign additional validation here isn't useful. It's also not really negative, just pointing out that it's unnecessary.

Reply:
So, should this be a CharField? If so, typically you need a maximum length. Any thoughts on what would be a good number for that?

Reply:
Ok well at least in the docs of Django I see online it looks like a SlugField's default length is 50. I'll go with that for now.